docs/modules/develop/pages/guide_migrate_webpacker_app.adoc

```diff
@@ -66,7 +57,7 @@ If it's not the case you must create it. Then, copy Decidim customizable assets
 
 * Copy the file https://github.com/decidim/decidim/blob/develop/decidim-generators/lib/decidim/generators/app_templates/decidim_application.js[decidim_application.js] to `app/packs/src/decidim/decidim_application.js`
 * Copy the file https://github.com/decidim/decidim/blob/develop/decidim-generators/lib/decidim/generators/app_templates/decidim_application.scss[decidim_application.scss] to `app/packs/stylesheets/decidim/decidim_application.scss`
-* Copy the file https://github.com/decidim/decidim/blob/develop/decidim-generators/lib/decidim/generators/app_templates/decidim-settings.scss[_decidim-settings.scss] to `app/packs/stylesheets/decidim/_decidim-settings.scss`
+* Copy the file https://github.com/decidim/decidim/blob/develop/decidim-core/app/packs/stylesheets/decidim/_decidim-settings.scss[_decidim-settings.scss] to `app/packs/stylesheets/decidim/_decidim-settings.scss`
```

Comment:
This file is in both places and it has the same contents, but I think that using the one from generators is more accurate, as it is the one that will be added to new applications.

Reply:
Ah, sorry!

That link is broken because it's missing the underscore. I thought the link is incorrect.

Reply:
Fixed the link.